spec/controllers/chapters_controller_spec.rb

```diff
@@ -9,7 +9,10 @@
   let(:unposted_work) { create(:work, authors: [user.pseuds.first]) }
   let(:banned_user) { create(:user, banned: true) }
   let(:banned_users_work) { create(:work, posted: true, authors: [banned_user.pseuds.first]) }
-
+  let(:chaptered_work) { create(:work_with_chapters, chapters_count: 3, posted: true) }
```

Comment:
We can remove `posted: true` here if the factory has it by default.

Reply:
Should we move these lets inside `describe "index"` so they don't get created for the blocks that don't use them?